src/constants.ts

```diff
@@ -7,3 +7,16 @@ export const RC_FILE_PATH = `${homedir()}/.volusionrc`;
 export const THUMBNAIL_PATH = "thumbnail.png";
 export const USER_DEFINED_BLOCK_CONFIG_FILE = "./dist/defaultConfig.json";
 export const ELEMENT_VERSION = packageFile.version;
+
+export const INTEGRATIONS: Record<string, { id: number } | undefined> = {
+    volt: {
+        id: 1,
+    },
+    // tslint:disable-next-line: object-literal-sort-keys
```

Comment:
**Suggestion:** You can get around this eslint rule by adding line breaks between keys that don't need to be sorted together.